torchqdynamics/mesolve.py

```diff
@@ -1,14 +1,23 @@
 from math import sqrt
+from typing import List, Optional
 
+import numpy as np
 import torch
 
-from .odeint import odeint
+from .odeint import AdjointQSolver, odeint
 from .solver import Rouchon
+from .solver_utils import inv_sqrtm, kraus_map
+from .utils import trace
 
 
 def mesolve(
-    H, jump_ops, rho0, tsave, solver=None, sensitivity='autograd', parameters=None
+    H, jump_ops, rho0, t_save, exp_ops: Optional[List[torch.Tensor]] = None,
+    solver=None, sensitivity=None, parameters=None
 ):
```

Comment:
Actually I'm thinking. Do we want the following function signature instead?
```python
def mesolve(H, jump_ops, rho0, t_save, *, exp_ops=None, solver=None, sensitivity=None, parameters=None)
```
(with the star between args and kwargs arguments)
Pytorch does it, it forces keyword-only arguments after the star. I think it's nice to avoid mistakes in the arguments ordering by users.